src/index.js

```diff
@@ -289,7 +298,7 @@ class Dropzone extends React.Component {
         onDragLeave={this.onDragLeave}
         onDrop={this.onDrop}
       >
-        {this.props.children}
+        {this.renderChildren()}
```

Comment:
Is that what you mean? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/static